static/src/js/containers/MapContainer/MapContainer.tsx

```diff
@@ -469,12 +450,8 @@ export const MapContainer: React.FC<MapContainerProps> = (props) => {
 
   // If the focusedCollectionId is set, get the added and removed granule ids
   if (focusedCollectionId && focusedCollectionId !== '') {
-    const { collections } = project
-    const { byId: projectById } = collections
-    const { [focusedCollectionId]: focusedProjectCollection = {} } = projectById
-
-    const { granules = {} } = focusedProjectCollection
-    const { addedGranuleIds = [], removedGranuleIds = [] } = granules
+    const { granules = {} } = focusedProjectCollection as ProjectCollection
```

Comment:
I did notice that if you remove all the colls from the project the map will still hang onto the last one that is true in prod though as well